content/en/blog/2024/contribute-as-sig-docs-reviewer.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: blog
+title: "From Code to Quill: Embark on a Legendary Kubernetes Quest with SIG Docs"
+date: 2024-05-16
+slug: contribute-as-sig-docs-reviewer
+author: "Ricardo Amaro (Acquia)"
+---
+
+You've probably come across the saying, "Contributing doesn't always mean writing lines of code"
+whispered to the wind or glimpsed in the margins of a sacred open-source tome.
+What is the truth behind this tale as old as time (or at least as old as Kubernetes itself)?
+
+Imagine Sarah: an enthusiast of open-source whose keyboard had seen more action crafting
+technical masterpieces for her projects than a knight's sword in battle. Sarah's had such
+a smooth relationship with Git, that she could `git commit` and `git push` in her sleep,
+and her etiquette in pull-requests was so polished, it could make a silver spoon look tarnished.
+
+But here's the twist: when it came to the mysterious _art-of-coding_, Sarah felt more
```

Comment:
Has Sarah signed off on this paragraph (and narrative in general)?  I'm worried that it overemphasizes her lack of programming skills.  If she learns to program next year, is this going to be embarassing?

Also, consider `_art of programming_` instead.  "coding" feels weird in this context.

Reply:
Sarah is a fictional character. We might write something more in the future, you are right. But up to now, this is the story to be unveiled further in the next chapters.

Reply:
Updating to `_art of programming_`. Thank you!

Reply:
I am opposed to writing this story with a fictional character, particularly one that's female.

Reply:
Removed the fictional character with a more inclusive "achievement journey" aimed directly at the reader.